src/index.html

```diff
@@ -22,6 +22,387 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
```

Comment:
The image source path './images/logo.png' should be verified to ensure the image is correctly located in the specified directory. If the image does not display, check the path and file name.